src/components/Section/Military/Disciplinary/Disciplinary.jsx

```diff
@@ -77,6 +82,10 @@ class Disciplinary extends Subsection {
   }
 
   render() {
+    const { formType } = this.props
+    const formTypeConfig = formType && formConfig[formType]
```

Comment:
@gidjin I'm not as concerned w this one since this prop comes from the Redux state (where there is a default form type set) via the `MilitaryConnector`